changes/5774.fixed

```diff
@@ -0,0 +1 @@
+Fixed the bug that required users and administrators to manage additional permission to be able to use saved views.
```

Comment:
Thinking about this more, let's make sure the saved-view docs clearly explain how permissions do and do not intersect with saved-views...

Reply:
Will include that with #5471